.github/workflows/e2e.yml

```diff
@@ -17,9 +17,13 @@
 
 on:
   pull_request:
+    paths-ignore:
+      - 'docs/**'
   push:
     branches:
       - dev
+    paths-ignore:
+      - 'docs/**'
```

Comment:
I remember I told you that this can not be added because if the workflow is skipped, the PRs cannot be merged

Reply:
Sorry,I forgot this. I'll see if i can fix it. If not i will remove it. @kezhenxu94

Reply:
I try to modify ```e2e-test```  with ```https://github.com/dorny/paths-filter#examples``` and ```https://github.com/dorny/paths-filter/issues/97```. Please check if this is correct. If not i will remove it. Thanks. @kezhenxu94

Reply:
It seems like ```dorny/paths-filter@v2 is not allowed to be used in apache/dolphinscheduler```. Sad...

Reply:
I have submitted a request to apache infra to see if this plugin can be enable. ```https://issues.apache.org/jira/browse/INFRA-23069?jql=project%20%3D%20INFRA%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20created%20DESC%2C%20priority%20DESC%2C%20updated%20DESC```

In the mean time, I will mark this pr as draft until this plugin is enable or i can find another way to solve this problem.

Reply:
@SbloodyS you can use this getsentry/paths-filter@v2 instead, seems they have the same functionalities and this is approved already by infra team as I know

Reply:
> @SbloodyS you can use this getsentry/paths-filter@v2 instead, seems they have the same functionalities and this is approved already by infra team as I know

Wow, It seems working. I will do more tests to verify it. Thanks.